scripts/docgen/main.ts

```diff
@@ -32,7 +32,10 @@ export const config: DocGenConfig = {
                             "static",
                             "generated"
                         ),
-                        transformRelativePath: (path) => `${basename(path)}.raw`
+                        transformRelativePath: (path) =>
+                            `${basename(path)}.raw`,
+                        transformJsImports: (snippet) =>
```

Comment:
This will just replace everything including ".js" in the file, including comments etc. Unlikely to matter in the near term, but eventually we'll want to integrate it with the transformers that occur as part of the snippet extraction process.